### ISSUE TITLE
Sleep issue delay wake up

### ISSUE THREAD

**Issue by bitoy81** [WhyToFly/SurfaceLaptop3-OpenCore#1]
Do you issue with delay wake up. I got like 20 sec delay before display shows up. Haven’t tried your efi since bigsurface kext stop working after aug firmware update. So don’t update your surface lol.

**Comment by WhyToFly** [comment]
Hi! It seems I am having some issues with waking up (sometimes the screen doesn't come on at all) so I usually shut the laptop down instead of closing the lid... Let me know if you end up figuring out what is causing the problem!
If this is also happening on boot for you try enabling "enable-dbuf-early-optimizer" in your GPU DeviceProperties (https://github.com/acidanthera/WhateverGreen/blob/master/Manual/FAQ.IntelHD.en.md#fix-the-issue-that-the-builtin-display-remains-garbled-after-the-system-boots-on-icl-platforms)

**Comment by bitoy81** [comment]
I don’t have that issue it’s just the delay. But now it’s worst since I can’t even use my keyboard and touchpad. So it’s just an ipad lol touchscreen only. Gonna wait till Microsoft fix the firmware or bigsurface kext gets updated.

**Comment by WhyToFly** [comment]
Oh, interesting... Thanks for the advice, I'll stay on this version then 😅 
Do you mind sharing your config? Maybe I merge some parts into mine to fix my wake issue

**Comment by bitoy81** [comment]
https://github.com/Xiashangning/BigSurface/issues/50

**Comment by bitoy81** [comment]
Check this post I attached my config. I’m still using Monterey so maybe the reason I don’t get those issues.

**Comment by bitoy81** [comment]
> Oh, interesting... Thanks for the advice, I'll stay on this version then 😅
> Do you mind sharing your config? Maybe I merge some parts into mine to fix my wake issue

Yes don’t run windows it may autoupdate 😅 your keyboard will show detachable once updated.

**Comment by WhyToFly** [comment]
Thanks! I'll check it out once I have some time. I'm not planning on running Windows anytime soon because I don't have a lot of space on my SSD so hopefully I won't run into this issue... But thanks for the warning and I hope your issue gets resolved soon!
